firmware/components/micropython/Kconfig

```diff
@@ -180,6 +180,13 @@ menu "MicroPython"
 				If SPIRAM is not used, heap is allocated from DRAM and setting the heap size too large
 				may result in insuficient heap for C services like mqtt, gsm, curl...
 
+		config MICROPY_TAKE_MOAR_HEAP
```

Comment:
Please use proper english descriptions for configuration options

Reply:
Fair enough